src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier ColorSupplier = new ColorSupplier();
+
+    public Ball GetRandomBall() {
+        ColorOfBall randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;
```

Comment:
Magic number '100' should be replaced with a named constant to give it meaningful context. For instance, you could define a constant like 'private static final int MAX_BALL_NUMBER = 100;' at the class level.